private/about.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
We should move this file under `imports/data` directory, and break it down to two files, 
The scope of the private directory is limited to server side only.

very roughly : 

`imports/data/flags.json` and `imports/data/contributors.json`
```
[
  {
     name: 'xyz',
     avatar: 'http://link or gravater',
     intro: 'lorem ipsum',
     socials:[
         {website: 'URL'},
         {twitter: '@handel'},
     ....
     ],
     type: 'code'
  },
   ....
]
```

on about page you can load this `json` files with their relative path
```
imports contributors from 'imports/data/contributors.json'
```
with the help of the helper function contributors can be passed to the template 
```
`contributors : function() { return contributors }`
```
and then we can iterate over it using 
`{{#each contributors}} {{/each}}` block.

To display contributors into their respective section we can use `equals` template helper  `{{#if equals type 'code'}}`

Reply:
I do not see imports/data directory in the project. Should I create it? thanks.

Reply:
Yes, `imports` directory is a part of a new directory structure suggested in meteor guide, our migration to the new directory structure is long over due.

Reply:
Noted. Will get this done before Sunday.

Reply:
@distalx  Please kindly review